<?xml version='1.0' encoding='UTF-8'?>
<rss xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/" version="2.0">
  <channel>
    <title>1+3t升级9.02以后，某个APP在北京联通4g下无法联网，WIFI下却可以</title>
    <link>https://bbs.saraba1st.com/2b/</link>
    <description>1+3t升级9.02以后，某个APP在北京联通4g下无法联网，WIFI下却可以</description>
    <docs>http://www.rssboard.org/rss-specification</docs>
    <generator>python-feedgen</generator>
    <lastBuildDate>Thu, 09 Jul 2020 18:45:01 +0000</lastBuildDate>
    <item>
      <title>1+3t升级9.02以后，某个APP在北京联通4g下无法联网，WIFI下却可以[0-50]</title>
      <link>https://bbs.saraba1st.com/2b/thread-1849342-1-1.html</link>
      <description>1+3t升级9.02以后，某个APP在北京联通4g下无法联网，WIFI下却可以&#13;
app里改IP直连也联网提示网络配置错误；但是今天却发现，私人DNS设置dns.rubyfish以后，该app 北京联通4g网络下无法联网的问题解决了。

why，是app和9.02不兼容，还是刚好北京联通4g网络ban了这个app？

我的WIFI网络也是北京联通啊。</description>
      <content:encoded><![CDATA[<p><b>seelestil: </b><br>
<span>1+3t升级9.02以后，某个APP在北京联通4g下无法联网，WIFI下却可以</span><br>
<span>app里改IP直连也联网提示网络配置错误；但是今天却发现，私人DNS设置dns.rubyfish以后，该app 北京联通4g网络下无法联网的问题解决了。</span><br>
<span>why，是app和9.02不兼容，还是刚好北京联通4g网络ban了这个app？</span><br>
<span>我的WIFI网络也是北京联通啊。</span><br>
</p><p><b>seelestil: </b><br>
<span>开了私人dns以后，百度贴吧等app却不能联网了</span><br>
<span>—— 来自 OnePlus ONEPLUS A3010, Android 9上的 S1Next-鹅版 v2.1.2</span><br>
</p>]]></content:encoded>
      <guid isPermaLink="false">1849342[0-50]</guid>
    </item>
  </channel>
</rss>
